tests/python/unittest/test_auto_scheduler_loop_state.py

```diff
@@ -142,6 +142,318 @@ def test_compute_at_root_inline():
     assert s0[conv].iters[5].range.extent == 7
     assert s0[conv].iters[6].range.extent == 7
 
+def test_follow_split_follow_fused_split():
```

Comment:
Put this function in the back of `test_cache_read_write`.
Have a careful check that there seems to be two `test_cache_read_write` function in this file? This should not be marked in the git diff.